recipes/google-resumable-media/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set version = "0.0.2" %}
```

Comment:
It looks like the newest version on pypi is `0.1.1`. Is there a reason to use the older version?